components/login/login.scss

```diff
@@ -0,0 +1,384 @@
+                        background-color: var(--center-channel-bg);
+                        color: rgba(var(--center-channel-text-rgb), 0.56);
+                        font-size: 12px;
+                        line-height: 0;
+                    }
+                }
+
+                .login-body-card-form-login-options {
+                    display: flex;
+                    flex: 1;
+                    flex-wrap: wrap;
+                    justify-content: space-between;
+                    column-gap: 8px;
+                    row-gap: 8px;
+
+                    .login-body-card-form-login-option {
+                        @include secondary-button;
+                        @include button-medium;
+
+                        min-width: 136px;
+                        flex: 1 1 0%;
+                        border-color: rgba(var(--center-channel-text-rgb), 0.16);
+                        color: rgba(var(--center-channel-text-rgb), 0.72);
+                        text-decoration: none;
+
+                        .login-body-card-form-login-option-label {
+                            margin-left: 7px;
+                        }
+                    }
+
+                    &.column {
+                        flex-direction: column;
+                        margin-top: 22px;
+                        row-gap: 24px;
+
+                        .login-body-card-form-login-option {
+                            flex: 1 0 auto;
+                        }
+                    }
+                }
+            }
+        }
+
+        .login-body-custom-branding-markdown {
+            flex: 1;
+            margin-top: 28px;
+
+            ul + p,
+            ol + p {
+                margin-top: 0.6em;
+            }
+
+            p + ul,
+            p + ol {
+                margin-top: 0.6em;
+            }
+
+            img {
+                max-width: 450px;
+            }
+
+            p {
+                width: 100%;
+                margin: 0;
+                color: rgba(var(--center-channel-text-rgb), 0.72);
+                font-size: 18px;
+                font-weight: 400;
+                line-height: 28px;
+                white-space: pre-wrap;
+                word-break: break-word;
+            }
+        }
+    }
+}
+
+@keyframes shake {
+    10%,
+    90% {
+        transform: translate3d(-1px, 0, 0);
+    }
+
+    20%,
+    80% {
+        transform: translate3d(2px, 0, 0);
+    }
+
+    30%,
+    50%,
+    70% {
+        transform: translate3d(-4px, 0, 0);
+    }
+
+    40%,
+    60% {
+        transform: translate3d(4px, 0, 0);
+    }
+}
+
+@media screen and (min-width: 1680px) {
+    .login-body .login-body-content {
+        .login-body-message,
+        .login-body-card,
+        .login-body-no-login {
+            width: 610px;
+        }
+
+        .login-body-message .login-body-message-title {
+            padding-right: 130px;
+        }
+    }
+}
+
+@media screen and (max-width: 1199px) {
```

Comment:
are these breakpoints use in the media queries constant across the application? I mean, are those the ones we use and should keep using? if so, wouldn't it make sense to have it as css variables?